template-aws-logs.yaml

```diff
@@ -141,6 +149,323 @@ Resources:
                 Resource: !GetAtt FirehoseLogStreams.Arn
       Description: Role for Subscription Filters (to write to Firehose)
 
+  SelfMonitoringDashboard:
+    Type: AWS::CloudWatch::Dashboard
+    Properties:
+      DashboardName: !Sub
+        - "DynatraceLogForwarder-${__Region__}-SelfMonitoring-${__StackName__}"
+        - __StackName__: !Ref 'AWS::StackName'
+          __Region__: !Ref 'AWS::Region'
+      DashboardBody: !Sub
+        - |
+          {
+              "widgets": [
+                  {
+                      "height": 6,
+                      "width": 12,
+                      "y": 6,
+                      "x": 12,
+                      "type": "metric",
+                      "properties": {
+                          "metrics": [
+                              [ "DT/LogsStreaming", "Kinesis record age", "function_name", "${__LambdaName__}", { "stat": "Minimum" } ],
+                              [ "...", { "stat": "Average" } ],
+                              [ "..." ]
```

Comment:
Shouldn't it be `{ "stat": "Average" }` here?

Reply:
If this is not specified then it's taken from higher level "stat", there is stat=Maximum 5 lines below